### ISSUE TITLE
Atelier creation

### ISSUE THREAD

**Issue by giuliarouge** [OpenDesign-SUPSI/GroupA-1415#3]
I propose each person to write 3 stuff for the atelier, then we can choose the best to be created in the lab.
Mine are a shelf (see example on furnitures folder), a bench/sofa and a  kitchen storage (place where we can store food and cutlery, tea and coffe)

**Comment by frokeneriksen** [comment]
Hi Giulia
I don't think the point of this project is for everyone to design their own furniture. (if that is what you mean with "write"). I think the point is for all 4 of us to work on the same project and develop one piece of furniture. Which is why I am asking if we have landed on one project - before anyone starts to design anything. I am up for doing any of the projects you guys have suggested.

**Comment by frokeneriksen** [comment]
And I think we should keep in mind that we are supposed to make the furniture in one day - so the smaller the better...

**Comment by francesca17** [comment]
yes i'm agreed with hanne, for this reason i think that something like a tool carrier is possible to make in one day and useful for all of everyone ( we let every time all tools on a table, i think it will nice for us have maybe lot of small tool carriers?!) maybe lots of tool carrier that when you put together they create a container or something similar, what do you think? :)

**Comment by soraiabinz** [comment]
I agree on the importance of creating something that can be feasibly produced in a day. But I also think another important consideration is to choose a project that can facilitate each member's contribution. 

For this reason I think it may be more collaborative to go for a storage unit, or a bench/seating with an integrated storage unit as Giulia suggested. Using this option for the project, each person can create a part of the project. ie. someone could create the bench part, another person could create the storage element, and another person could create some supplementary compartments (for cutlery or tools) within the storage areas. 

This is why I think it may be a bit tough to have everyone in our group take part of the production of a tool box.

Of course though, if we do create a larger piece of furniture, it shouldn't be to elaborate or complicated, so we can manage to produce it in a day.

**Comment by soraiabinz** [comment]
Otherwise, if you want to stick to the tool box, or portable container idea that francesca suggested - we could perhaps do something kind of like this(attached) that we can just move around in the class to wherever we need it.

![image](https://cloud.githubusercontent.com/assets/9048838/5644140/affa9182-9660-11e4-8279-cc3b32760c0e.png)

**Comment by giuliarouge** [comment]
let's create a storage unit!

**Comment by frokeneriksen** [comment]
I am voting for a toolbox with a twist like Soraia's latest contribution:)

**Comment by francesca17** [comment]
cool!! i'm agreed me too :)

**Comment by soraiabinz** [comment]
I'm not sure I understand who you're agreeing with francesca..?

**Comment by soraiabinz** [comment]
ok tool box it is
